packages/manager/tools/testcafe/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@ovh-ux/testcafe-manager",
+  "version": "0.0.0",
+  "private": true,
+  "description": "Testcafé files for OVH Manager.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ovh/manager.git",
+    "directory": "packages/manager/tools/testcafe"
```

Comment:
Did you have consider using a different name for the folder/package name?

`testcafe` seems to be to generic for me but I don't have other inputs for now.

Reply:
nop, from my PoV, because the package is named "testcafe-manager", it's ok to name it "testcafe".
But I can put something else if you want.

Reply:
As you mentioned, we already prefixed the package name so I'm ok to keep `testcafe` as folder name.